src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import StyledTabs from './Tabs.styled';
+
+export default function Tabs(props) {
+	const [content, setContent] = useState('');
+
+	console.log(content);
+
+	const clickHandler = (e) => {
+		setContent(e.target.dataset.content);
```

Comment:
W React raczej opieramy się na `state` lub `props`, a nie DOM.
Jeśli wykorzystać rozwiązanie, o którym piszę u góry to będzie to łatwiejsze do zastosowania ;)